packages/pwa-kit-dev/package.json

```diff
@@ -106,8 +105,6 @@
     "source-map-loader": "^4.0.1",
     "speed-measure-webpack-plugin": "^1.5.0",
     "svg-sprite-loader": "^6.0.11",
-    "ts-loader": "^9.5.1",
-    "tsx": "^4.16.2",
```

Comment:
We no longer used these, is this correct?

Reply:
Correct. I think these were from the original server side extensibility implementation